src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidationContext.cs

```diff
@@ -69,6 +71,12 @@ public class OpenIdConnectProtocolValidationContext
         /// <summary>
         /// This id_token is assumed to have audience, issuer, lifetime and signature validated.
         /// </summary>
+        [Obsolete("The 'ValidatedIdToken' property is obsolete. Please use 'ValidatedJsonWebToken' instead.")]
```

Comment:
Perhaps we can explain here that ValidatedIdToken is related to IdTokenValidator and that ValidatedJsonWebToken/JsonWebTokenValidator combination should be used going forward.

Reply:
Good idea. :) I added this explanation on the IdTokenValidator obsolete attribute, but I can put it in this obsolete attribute as well.